inventories/broker.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import yaml
+import os
+import subprocess
+import sys
+
+try:
+    from StringIO import StringIO  # pyright: reportMissingImports=false
+except ImportError:
+    from io import StringIO  # pyright: reportMissingImports=false
+
+from collections import defaultdict
+
+
+try:
+    DEVNULL = subprocess.DEVNULL
+except AttributeError:
```

Comment:
It was added in Python 3.3 so if you make the script Python 3 only then I think you're guaranteed that it's present. See https://docs.python.org/3/library/subprocess.html#subprocess.DEVNULL